app/guide/structs.vmd

```diff
@@ -222,11 +222,14 @@ struct Spaceship imm {
 
 fn main() export {
   ship = Spaceship("Serenity", 2);
+  ship2 = ship;
   println(ship.numWings);
+  println(ship2.numWings);
 }
 ```
 ```stdout
 Spaceship("Serenity", 2)
+Spaceship("Serenity", 2)
```

Comment:
You're right that it will print 2 lines, and now that I'm looking at it, it will probably print:
2
2
since we're just printing the numWings

Reply:
of course...